.github/workflows/wheels.yml

```diff
@@ -0,0 +1,38 @@
+name: Build
+
+on:
+  schedule:
+    # Run weekly on Monday 00:00
+    - cron:  '00 00 * * MON'
+  push:
+    branches: [rel-*]
+  pull_request:
+    branches: [rel-*]
+  workflow_dispatch:
+
```

Comment:
It is so nice and simple. I hope onnxruntime's packaging pipeline could be so simple.